### PR TITLE
Closes #143: Manage inventory access requests

### LINKED ISSUE #143: Product Inventory: Manage Access Requests
Sharing the same modal as permissions https://github.com/MobilityLabs/pdredesign-server/issues/142, we show a list of all access request to this inventory so access can be either granted or rejected.

### PR DESCRIPTION
**Pull request by bithavoc**

### CODE REVIEW

File: app/assets/javascripts/client/inventories/ManageInventoryAccessRequestsCtrl.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+      InventoryAccessRequest.update(params, { status: status })
+      .$promise
+      .then(function() {
+        vm.loadList();
+      });
+    }
+
+    vm.denyRequest = function(id) {
+      if (confirm("Are you sure you want to deny this access request?")){
```
Comment (MakotoTheKnight) [inline app/assets/javascripts/client/inventories/ManageInventoryAccessRequestsCtrl.js]: To make this more testable, inject `$window` and then perform `$window.confirm`.

File: app/assets/javascripts/client/inventories/manage_inventory_access_requests.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+        <div class='col-sm-2'>
+          <p class='name_label'>{{requester.full_name}}</p>
+        </div>
+        <div class='col-sm-3'>
+          <p class='info'>{{requester.email }}</p>
+        </div>
+        <div class='col-sm-3'>
+          <div class="pull-left permission_transition">
+            <span class="old_level">{{humanPermissionName(requester.current_permission_level.human)}}</span>
```
Comment (MakotoTheKnight) [inline app/assets/javascripts/client/inventories/manage_inventory_access_requests.html]: Where's `humanPermissionName` defined at?  I'm not seeing it in the controller that's driving this view.

File: spec/javascripts/inventories/InventoryAccessRequestSpec.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    inject(function($resource, $rootScope, InventoryAccessRequest, $injector) {
+      scope = $rootScope.$new();
+      subject = InventoryAccessRequest;
+      $httpBackend = $injector.get('$httpBackend');
+    });
+  });
+
+  it("updates access request", function(){
+    $httpBackend.expectPATCH('/v1/inventories/10/access_requests', { status: 'denied' }).respond(200, {});
+    var users = subject.update({
```
Comment (MakotoTheKnight) [inline spec/javascripts/inventories/InventoryAccessRequestSpec.js]: You don't do anything with this variable.  If this is meant to be used as a verification bit and we're relying on `$httpBackend` to help with that, we probably don't need the variable.

**Comment by MakotoTheKnight** [comment]
The modal created from this should be larger; the size being used isn't large enough to allow for comfortable viewing of the buttons.

**Comment by MakotoTheKnight** [comment]
Go ahead and rebase this against staging and merge it in - we'll be able to use this in #144.

**Comment by bithavoc** [comment]
@MakotoTheKnight ready

**Pull request merged**